app/src/main/java/org/simple/clinic/appupdate/AppUpdateNotificationWorker.kt

```diff
@@ -51,13 +50,14 @@ class AppUpdateNotificationWorker(
     private const val NOTIFICATION_CRITICAL = 6
     private const val NOTIFICATION_CHANNEL_NAME = "Updates"
 
-    fun createWorkRequest(userClock: UserClock, schedule: AppUpdateNotificationSchedule): PeriodicWorkRequest {
+    fun createWorkRequest(userClock: UserClock, schedule: AppUpdateNotificationSchedule): OneTimeWorkRequest {
```

Comment:
It looks like we are using `OneTimeWorkRequest` but are not rescheduling it once it's done.

Reply:
oh, good catch.